cfpb/consumerfinance.gov
# TDP prototype changes

This code brings the current TDP activities search prototype on beta into main (with the exception that the code to falsely limit the activities to 75 has been reverted).

This doesn't include any backend simplification, which will come separately.

Testing:
  - Pull and build
  - Search and filters should work
  - Things should look like https://beta.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/teach/activities/
